c7n/resolver.py

```diff
@@ -36,7 +37,7 @@ def resolve(self, uri):
             # TODO: in the case of file: content and untrusted
             # third parties, uri would need sanitization
             fh = urlopen(uri)
-            contents = fh.read()
+            contents = fh.read().decode('utf-8')
```

Comment:
interesting, this fix would have resolved quite a few minor regressions we've seen on trunk with the py3 work as is.